ClinGen/clincoded
# Connects to #269 ft method field

This is a one text-string change to all method panels, Create/Edit and View, for Group, Family, and Individual

##### Testing Method

1. Create a Group, Family, and/or Individual.
2. Before saving, scroll down to the Method panel. Verify one item either at the bottom or second from the bottom (depends on the evidence type) has the title “Description of genotyping method” instead of the previous “Description of method(s).”
3. Save any/all of these, then view them. Verify that the corresponding field in the Methods panel of each also says “Description of genotyping method.”
